src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHostUserInterface.cs

```diff
@@ -1580,8 +1580,8 @@ private string ReadLineFromConsole(bool endOnTab, string initialContent, bool ca
             }
 
 #endif
-                do
-                {
+            do
+            {
 #if UNIX
                     keyInfo = Console.ReadKey(true);
```

Comment:
shouldn't this be moved left?

Reply:
Yes, but CodeFormatter follows parsing - we parse on Windows and UNIX const does not set. So CodeFormatter doesn't format Unix code at all in whole code base. I suggest ignore this in the PR.

Reply:
ignore meaning revert?

Reply:
No, we don't format UNIX code so we have nothing to revert.

Reply:
I see.